common/util.go

```diff
@@ -477,19 +477,23 @@ func CreateHistoryStartWorkflowRequest(
 	now time.Time,
 ) *historyservice.StartWorkflowExecutionRequest {
 	histRequest := &historyservice.StartWorkflowExecutionRequest{
-		NamespaceId:            namespaceID,
-		StartRequest:           startRequest,
-		ContinueAsNewInitiator: enumspb.CONTINUE_AS_NEW_INITIATOR_WORKFLOW,
-		Attempt:                1,
-		ParentExecutionInfo:    parentExecutionInfo,
+		NamespaceId:              namespaceID,
+		StartRequest:             startRequest,
+		ContinueAsNewInitiator:   enumspb.CONTINUE_AS_NEW_INITIATOR_UNSPECIFIED,
+		Attempt:                  1,
+		ParentExecutionInfo:      parentExecutionInfo,
+		FirstWorkflowTaskBackoff: backoff.GetBackoffForNextScheduleNonNegative(startRequest.GetCronSchedule(), now, now),
 	}
 
 	if timestamp.DurationValue(startRequest.GetWorkflowExecutionTimeout()) > 0 {
 		deadline := now.Add(timestamp.DurationValue(startRequest.GetWorkflowExecutionTimeout()))
 		histRequest.WorkflowExecutionExpirationTime = timestamp.TimePtr(deadline.Round(time.Millisecond))
 	}
 
-	histRequest.FirstWorkflowTaskBackoff = backoff.GetBackoffForNextScheduleNonNegative(startRequest.GetCronSchedule(), now, now)
+	if len(startRequest.CronSchedule) != 0 {
```

Comment:
I personally prefer 
```suggestion
	if startRequest.CronSchedule != "" {
```
but it is a matter of preferences. Funny fact, that compiler produces exact the same code for both cases (read it somewhere on SO from go lang authors).

Reply:
seems contentious. will leave as len since we do this for validating cronschedule in other parts of the code base and because there is no consensus on which is better